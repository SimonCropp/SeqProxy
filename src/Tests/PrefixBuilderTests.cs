using System;
using ApprovalTests;
using Xunit;
using Xunit.Abstractions;

public class PrefixBuilderTests :
    XunitLoggingBase
{
    [Fact]
    public void Build()
    {
        var builder = new PrefixBuilder("TheAppName", new Version(1, 2));
        var build = builder.Build(ClaimsBuilder.Build(), "theUserAgent");
        Approvals.Verify(build,s => s.Replace(Environment.MachineName,"TheMachineName"));
    }

    public PrefixBuilderTests(ITestOutputHelper output) :
        base(output)
    {
    }
}