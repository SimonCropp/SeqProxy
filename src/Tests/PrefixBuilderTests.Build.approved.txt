{'AppName':'TheAppName','AppVersion':'1.2','Server':'MACPRO','Claims':{'claimType':'superadmin'},'UserAgent':'theUserAgent',